### ISSUE TITLE
Fake It Until You Become It

### ISSUE THREAD

**Issue by farosato** [farosato/fabiorosatodotcom#31]
---
slug: fake-it-become-it
description: Here's how you build confidence even though you're overcome with fear, self-doubt, or uncertainty.
---
We tend to view performance as a one-way street.
We often hear about a physically gifted athlete who underperforms on the field or a smart student who flounders in the classroom.
The typical narrative about underachievers is that if they could just _“get their head right”_ and develop the correct _“mental attitude”_ then they would perform at the top of their game.

There is no doubt that your mindset and your performance are connected _in some way_.
But this connection **works both ways**.
A confident and positive mindset can be both the **cause** of your actions and the **result** of them.

**The link between physical performance and mental attitude is a two-way street.**

> Your beliefs inspire your actions.
> And your actions inspire your beliefs.

These aren't just feel-good notions or fluffy self-help ideas.
There is hard science proving the link between behavior and confidence.
Amy Cuddy, a Harvard researcher who studies body language, has shown through her research that simply [standing in more confident poses can increase confidence and decrease anxiety](https://journals.sagepub.com/eprint/CzbNAn7Ch6ZZirK9yMGH/full).

Subjects experienced actual biological changes in their hormone production including increased testosterone levels (which is linked to confidence) and decreased cortisol levels (which are linked to stress and anxiety).

These findings go beyond the popular _"fake it until you make it"_ philosophy.
According to Cuddy, you should _**“fake it until you become it.”**_

So, here's how to build confidence:

> Ask yourself, "What would the type of person you want to be do?"

For example, if you want to lose weight and be healthy, repeatedly ask yourself _“What would a healthy person do?”_

You may not feel like one, but if you act like a healthy person, then eventually you'll become one.

Confidence is a wonderful thing to have.
But if you find yourself overcome with fear, self-doubt, or uncertainty, then **let your behavior drive your beliefs**.

Play as if you’re at your best.
Work as if you’re on top of your game.
Talk to that person as if you’re confident.

You can **use bold actions to trigger a bold mindset**.

In short, ask yourself _**"What would a brave person do?"**_
